performance/front/table-and-chart-with-good-performance.md

```diff
@@ -0,0 +1,41 @@
+# A table or a chart on my project takes to much time to render
+
+## Owner: [Justine Mignot](https://github.com/justinemignot)
+
+## Do you need to display all data you are displaying currently ?
+
+### Chart (Owner: [Thibaut Guedou](https://github.com/thibautguedou3))
+
+If your user only needs 50 points, do not display 500!
+Ask yourself this question: “which data do I need to display?”. If the data you receive from your API call or your SQL request contains 200 points, it does not necessarily means that you have to display these 200 points.
```

Comment:
I think it should be put in the front folder. Take a look at my PR on the repo: https://github.com/bamlab/dev-standards/pull/142

I think it is one of the point i have added react-native-performance.s.md

Reply:
This is in front folder, do you think I should move it ? The file won't have sense if I move this part ?